File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest41536.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest41536 {

    @PostMapping(path="/BenchmarkTest41536", consumes="text/plain")
    public void BenchmarkTest41536(@RequestBody String rawBody, HttpServletRequest request, HttpServletResponse response) throws Exception {
        String rawData = rawBody != null ? rawBody : "";
        String data = rawData.isEmpty() ? "default" : rawData;
        Integer.parseInt(data);
        response.setContentType("application/json");
        response.getWriter().print("{\"id\":0}");
    }
}

File: Benchmarks/enterprise/python/flask/testcode/benchmark_test_41536.py
# SPDX-License-Identifier: Apache-2.0
import os
from urllib.parse import unquote
from flask import request, jsonify


def BenchmarkTest41536():
    user_id = request.args.get('id', '')
    data = unquote(user_id)
    os.system('echo ' + str(data))
    return jsonify({"result": "success"})
